<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JAVShell设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>系统设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>启动选项</string>
         </property>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>371</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>开机自动启动</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>381</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>自动登录Windows</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>251</width>
            <height>81</height>
           </rect>
          </property>
          <property name="title">
           <string>登录信息</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>20</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>50</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>20</y>
             <width>54</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>50</y>
             <width>54</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </widget>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>381</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>自动登录JAV</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>210</y>
            <width>251</width>
            <height>81</height>
           </rect>
          </property>
          <property name="title">
           <string>登录信息</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>20</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>50</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>20</y>
             <width>54</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>50</y>
             <width>54</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>存储位置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>存储位置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>录像路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="add_dirbtn">
               <property name="text">
                <string>增加</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="del_dirbtn">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>抓拍路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item>
              <widget class="QPushButton" name="capturebtn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>下载路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
             <item>
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
             <item>
              <widget class="QPushButton" name="downloadbtn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savebtn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="canclebtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_start">
   <property name="text">
    <string>启动</string>
   </property>
   <property name="toolTip">
    <string>启动JAVClient</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止JAVClient</string>
   </property>
  </action>
  <action name="action_config">
   <property name="text">
    <string>显示配置</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出JAVShell</string>
   </property>
  </action>
  <action name="action_restart">
   <property name="text">
    <string>重新启动</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于JAVShell</string>
   </property>
   <property name="toolTip">
    <string>关于JAVShell</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/sharenet.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>savebtn</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>canclebtn</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
